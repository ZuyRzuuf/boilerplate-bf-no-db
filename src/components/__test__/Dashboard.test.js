import React from 'react';

import Dashboard from '../Dashboard';

describe('A suite', function() {
    test('renders isolated component correctly', () => {
        const wrapper = shallow(
            <Dashboard />
        );
        expect(wrapper).toMatchSnapshot();
    });

    test('renders components tree correctly', () => {
        const wrapper = mount(
            <Dashboard />
        );
        expect(wrapper).toMatchSnapshot();
    });

    test('renders HTML tree correctly', () => {
        const html = mount(
            <Dashboard />
        );
        expect(html).toMatchSnapshot();
    });
});
