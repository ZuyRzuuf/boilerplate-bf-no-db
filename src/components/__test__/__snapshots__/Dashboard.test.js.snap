// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A suite renders HTML tree correctly 1`] = `
<Dashboard>
  <div>
    Dashboard
  </div>
</Dashboard>
`;

exports[`A suite renders components tree correctly 1`] = `
<Dashboard>
  <div>
    Dashboard
  </div>
</Dashboard>
`;

exports[`A suite renders correctly 1`] = `
<Fragment>
  <div>
    Dashboard
  </div>
</Fragment>
`;

exports[`A suite renders isolated component correctly 1`] = `
<Fragment>
  <div>
    Dashboard
  </div>
</Fragment>
`;
